### DISCUSSION TITLE
sop/gh-pages

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#292] [Announcements]
# sop/gh-pages

快速完成博客的发布部署到 GitHub/Gitee Pages。

https://theme.sugarat.top/sop/gh-pages.html

**Comment by SuZhelevel6** [comment]
请问我这边部署到Github Pages的时候会丢失掉所有的样式，这可能是什么原因啊？

**Reply by ATQQ** [reply]
你看第三步 ③ 仔细检查一下是不是没配对

**Reply by SuZhelevel6** [reply]
是的，base配错了，感谢糖哥~

**Comment by jimmy-xyz** [comment]
嗯嗯对的
